_translator-files/po/es/Include-Client-Commands.po

```diff
@@ -6,28 +6,27 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2022-02-12 21:19+0100\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-07-20 12:34+0200\n"
 "Last-Translator: \n"
 "Language-Team: \n"
 "Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 3.0\n"
+"X-Generator: Poedit 3.1.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:2
-#, fuzzy, no-wrap
-#| msgid "`-M` or `--mutestream`  Starts Jamulus in muted state                                                      \n"
+#, no-wrap
 msgid "`-M` or `--mutestream`  Prevent others on a server from hearing what I play                                                      \n"
-msgstr "`-M` o `--mutestream`  Inicia Jamulus en estado silenciado                                                      \n"
+msgstr "`-M` o `--mutestream`  Evita que otros en el servidor escuchen lo que yo toco                                                      \n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:3
-#, fuzzy, no-wrap
-#| msgid "`--mutemyown`  Mute me in my personal mix (\"headless\" mode only)                                                      \n"
+#, no-wrap
 msgid "`--mutemyown` Prevent me from hearing what I play in the server mix (headless only)                                                      \n"
```

Comment:
It looks like there needs to be some checking for trailing spaces here (and in general).